src/FileSystem/FilesFinder.php

```diff
@@ -39,6 +39,7 @@ public function findInDirectoriesAndFiles(array $source, array $suffixes = [], b
 
         // filtering files in files collection
         $filteredFilePaths = $this->fileAndDirectoryFilter->filterFiles($filesAndDirectories);
+        $filteredFilePaths = array_map($this->filePathHelper->absolutePath(...), $filteredFilePaths);
```

Comment:
Just change to `realpath()` is enough:

```suggestion
        $filteredFilePaths = array_map(realpath(...), $filteredFilePaths);
```